cocotb/__init__.py

```diff
@@ -128,6 +128,14 @@ def _reopen_stream_with_buffering(stream_name):
 _library_coverage = None
 """ used for cocotb library coverage """
 
+top = None  # type: SimHandleBase
+"""
+A handle to the :make:var:`TOPLEVEL` entity. ``None`` if :mod:`cocotb` was not loaded from a simulator.
```

Comment:
I think @cmarqu is right. It's not linking in the docs, but works in the newsfragment.
```suggestion
A handle to the :envvar:`TOPLEVEL` entity. ``None`` if :mod:`cocotb` was not loaded from a simulator.
```

Reply:
Whoops missed that too.